src/Tools/ExternalAccess/Razor/Features/RazorEndpointAttribute.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Composition;
+using Microsoft.CommonLanguageServerProtocol.Framework;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.Razor.Features;
+
+[MetadataAttribute]
+internal class RazorEndpointAttribute : LanguageServerEndpointAttribute
```

Comment:
Could `RazorMethodAttribute` move to the shared layer?

UPDATE: Oh, I guess that would be a break because the namespace is wrong. I guess we'll just have to move over to use this in cohosting, so we can eventually remove RazorMethodAttribute, and move this to the shared project?

Reply:
I think we'll eventually want cohosting on the `Features` layer only right? If so I'll just leave this as is for now